samples/nrf9160/lwm2m_client/src/events/location_event_handler.c

```diff
@@ -26,13 +26,11 @@ static struct lwm2m_ctx *client_ctx;
 #define REQUEST_WAIT_INTERVAL K_SECONDS(5)
 
 #include <zephyr/logging/log.h>
-LOG_MODULE_REGISTER(lwm2m_location_event_handler, CONFIG_LWM2M_CLIENT_UTILS_LOG_LEVEL);
+LOG_MODULE_REGISTER(location_event_handler, CONFIG_LWM2M_CLIENT_UTILS_LOG_LEVEL);
```

Comment:
`CONFIG_LWM2M_CLIENT_UTILS_LOG_LEVEL` should also be changed.

And it seems that ipso/ipso_colour_sensor.c uses `CONFIG_LWM2M_LOG_LEVEL` for the default compile time logging level, that's probably not intentional?

It seems that this sample uses `LOG_MODULE_REGISTER()` in each file and doesn't use `LOG_MODULE_DECLARE()` at all. In bigger applications splitting the logging for different modules is probably a good idea, but maybe here it's a bit excessive. Usually, samples register just a single logging module for the whole sample, but this has 26 different logging modules. Just something to consider, I don't say it needs to be changed in this PR.

Reply:
True, actually it can be removed as the `location_event_handler.c` doesn't even have logging ATM.
Thanks for the input. I'll create a new ticket for the logging task as it isn't the focus in this PR.